stl/inc/xstring

```diff
@@ -2642,6 +2642,44 @@ private:
     }
 
 public:
+#if _HAS_CXX20
+    void _Move_assign_from_buffer(_Elem* const _Right, const size_type _Size, const size_type _Res) {
+        // Move assign from a buffer, used by basic_stringbuf
+        _Tidy_deallocate();
+        _Right[_Size]  = _Elem();

```

Comment:
I think this needs to use _Traits::assign like the other places adding a null terminator.